libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -229,6 +241,128 @@ describe('CalendarComponent', () => {
     ).toEqual('M T W T F S S');
   });
 
+  describe('active month', () => {
+    let todayDate: Date;
+
+    beforeEach(() => {
+      todayDate = new Date(2021, 0, 1);
+      spectator.setInput('todayDate', todayDate);
+    });
+
+    describe('when `minDate` is set', () => {
+      let minDate: Date;
+
+      beforeEach(() => {
+        minDate = new Date(2017, 0, 1);
+        spectator.setInput('minDate', minDate);
+      });
+
+      describe('when `minDate` changes to a later date', () => {
```

Comment:
This should perhaps be specified with the following `'when `minDate` changes to a later date than todayDate'`, since the test will only pass if we actually move the active month, which will be what todayDate is under the provided circumstances.

Reply:
Do it if you think it's better (instead of creating a very long comment 😂 )

Reply:
![](https://media.giphy.com/media/6w7tYtag2oZCJjKlkl/giphy-downsized.gif)